build/make_bundle.sh

```diff
@@ -5,7 +5,7 @@
 set -e
 
 VERSION=`grep "Version.*=.*\".*\"" version/version.go | sed "s,.*Version.*=.*\"\(.*\)\".*,\1,"`
-OLD_VERSIONS="v1beta1 v1alpha3 v1alpha2"
+#OLD_VERSIONS="v1beta1 v1alpha3 v1alpha2"
```

Comment:
Should this line be removed?

Reply:
> Seems like all the changes on CRD files are due to the yq version. Wondering if you are using the yq version 3.3.x, by any chance? yq 3.3.x output is not uniform across the Linux distributions, so we stuck with 3.2.x.

Regarding your comment @sgontla the CRD changes occur with yq v3.2.1, they are due to running 
```
operator-sdk generate crds
```
with Operator SDK version 0.18.2.